storage/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "robocorp-storage"
+version = "0.1.0"
+description = "Robocorp Asset Storage library"
+authors = [
+	"Cosmin P. <cosmin@robocorp.com>",
+]
+readme = "README.md"
+packages = [{include = "robocorp", from="src"}]
+
+[tool.black]
+skip-string-normalization = false
+skip-magic-trailing-comma = false
+
+[tool.isort]
+profile = "black"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+requests = "^2.28.2"
+tenacity = "^8.0.1"
```

Comment:
I'm not exactly sure what you have in mind, but `robocorp-tasks` shouldn't really need `requests` nor `tenacity`.

Maybe a better place would be `robocorp.http` for such common bits (and then this one and work items would rely on it?)

Reply:
We discussed this with @cmin764 ealier today over Zoom, and the point was to have a place for common code that deal with Control Room API requests / environment variables / etc. So mostly work items, asset storage, and process library (at some point).

We don't need to do it immediately, maybe when we need to change the copy-pasted bits it would make sense to harmonize, but it does have a fairly large maintenance overhead. The naming is up in the air, but actually I do like your suggestion of having it also in the public parts of robocorp-http. Then the requests wrapper would actually deliver some proper value.

Reply:
Maybe it could be used in `robocorp-vault` too... 

I think I saw a few tests failing due to internal errors (500) which could probably be fixed by a retry (https://github.com/robocorp/robo/actions/runs/5231372742/jobs/9445474509)

Reply:
Yes, it's about all our Control Room interactions (API requests, env vars handling) bundled together into a common package. We can call it any of the following:
- `robocorp-commons`
- `robocorp-utils`
- `robocorp-core`
- `robocorp-<name>`

What's your suggestion? (ofc, outside of this PR)

Reply:
I'm a bit hesitant on making a very big "common" library, since it's very hard to keep the API for something like that stable, which makes the maintenance a real pain. The most imporant reusable code (in my mind) is the Control Room API parts, so HTTP requests with retrying and some specific error handling. @fabioz proposed having them as user-facing functionality within `robocorp-http`, which I think could be a good idea, but haven't thought about the specifics.

Things like shared testing code or other non-production things should be inside the `devutils` dependency.

Reply:
Makes sense, thank you! We can discuss more once we decide to refactor the commons into a single source of truth. I'll mark this as "Resolved" since we came to a conclusion.